hack/test-e2e.sh

```diff
@@ -6,10 +6,10 @@ go clean -testcache
 
 # create ns if not exists because the `apply -f -` won't complain on a no-op if the ns already exists.
 kubectl create ns $KAPPCTRL_E2E_NAMESPACE --dry-run=client -o yaml | kubectl apply -f -
-go test ./test/e2e/kappcontroller -timeout 60m -test.v $@
+go test ./test/e2e/kappcontroller -timeout 60m -test.v $@ | tee tmp/e2eoutput.txt
```

Comment:
lets rm this file first then

Reply:
@cppforlife we can,  but i thought it was graceful enough to overwrite it each time.  I don't mind making the change but i'd appreciate understanding why you think that's better?

Reply:
> graceful enough to overwrite it each time

doesnt tee just keep on appending? if it does indeed override, then your 2nd tee would always blow away first tee.

Reply:
> doesnt tee just keep on appending

second one has a `-a`[ flag ](https://man7.org/linux/man-pages/man1/tee.1.html). default behavior is overwrite. 

I'll admit i didn't test very thoroughly and i'm just trusting the man pages.